contents/handbook/engineering/incidents.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Handling an incident
```

Comment:
Note: we have already runbook to deal with incidents https://github.com/PostHog/product-internal/tree/main/infrastructure/runbooks we should likely consolidate those two.

Reply:
Done!

Reply:
Did this happen?

Reply:
I'm sorting this out once this PR has merged 👍